lms/templates/main.html

```diff
@@ -61,7 +61,11 @@
   <link rel="icon" type="image/x-icon" href="${static.url(static.get_value('favicon_path', settings.FAVICON_PATH))}" />
 
   <%static:css group='style-vendor'/>
-  <%static:css group='${self.attr.main_css}'/>
+  % if uses_bootstrap:
+    <link rel="stylesheet" href="${static.url(self.attr.main_css)}" type="text/css" media="all" />
```

Comment:
Not sure if we have documentation anywhere that specifies that to use the pattern library / bootstrap that you must pass the variable into the template context - if not we should find a good place to add it.